section1/set8/nestedLoops/challenge.go

```diff
@@ -0,0 +1,32 @@
+package main
```

Comment:
I would appreciate if you include the goal of your problem; like what you are trying to achieve; also please what kind of i/p and o/p is expected from the program; just copy the problem statement here

Reply:
fixed